recipes/cairo/all/test_package/conanfile.py

```diff
@@ -4,7 +4,7 @@
 
 class TestPackageConan(ConanFile):
     settings = "os", "compiler", "build_type", "arch"
-    generators = "cmake"
+    generators = "cmake", "pkg_config"
```

Comment:
```suggestion
    generators = "cmake", "pkg_config"
```

pkg_config is useless here